code/game/machinery/computer/communications.dm

```diff
@@ -950,7 +950,10 @@ GLOBAL_VAR_INIT(cops_arrived, FALSE)
 
 			if(num_agents_created <= 0)
 				// We failed to run any midround sleeper agents, so let's be patient and run latejoin traitor
-				dynamic.picking_specific_rule(/datum/dynamic_ruleset/latejoin/infiltrator, forced = TRUE, ignore_cost = TRUE)
+				// SKYRAT EDIT START
+				//dynamic.picking_specific_rule(/datum/dynamic_ruleset/latejoin/infiltrator, forced = TRUE, ignore_cost = TRUE)
```

Comment:
This should probably still pull a bad guy in because it's not RNG, it's an action taken by a player

Reply:
At best I'm gonna make it a syndicate portal storm or something; I'm not that concerned with it overall because it's an edge case of a 25% chance of an objective that nobody (no progtots and ninjas don't get it) can get

Reply:
legit